### ISSUE TITLE
configuration of TEST $120/TEST $120 Special $90 with all attributes modifying pricing

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#5642]
While in the world of pain that is DPU, looking at it's handling of attributes, these products appear to be invalid/not real-world examples.
The base price is 120 but then all the attributes have a +something value. 
![image](https://user-images.githubusercontent.com/4391026/224539674-6dca382c-b313-43a6-8273-5e3184a4506d.png)

So the base price does not equate to any product at all.

I believe at least one colour and size should be +0 and optionally be set to default, no?

I think the test products should always be valid unless intended/marked as deliberately invalid to test handling.

**Comment by scottcwilson** [comment]
> I believe at least one colour and size should be +0 and optionally be set to default, no?

No.   You have to set a size and color on the product info page, and there's no guarantee that any choice will be more expensive than any other option.

**Comment by torvista** [comment]
I don't understand your answer. 
I know you have to choose an option on the product page.

My understanding is that a product should be either
 - no base price, and each attribute is the total price for each variant.
 - a base price for a default variant, and each alternate attribute modifies +/- that price for other variants.

**Comment by lat9** [comment]
In a _real_ store, I would be put off by the product's pricing as the product has a non-zero base price and then a required "Size" that further adds to the product's price.  That "Size" attribute, while not required by its configuration, _is_ essentially required since it's a dropdown-type attribute that subsequently forces a price-adding selection.

While I wouldn't suggest using the product's pricing structure in a real store, I don't feel that the pricing for this _example_ product is invalid.
